### PR TITLE
Add description, keywords meta tags

### PR DESCRIPTION
**Pull request by annagav**
#### What are the relevant tickets?
Related to #1112,
Using text from #1140
#### What's this PR do?
Added `description` and `keywords` meta tags to 'home_page' and 'program_page'.
Changed `h1` header to match the title of the homepage.

### CODE REVIEW

File: cms/templates/cms/program_page.html [hunk 1]
```diff
@@ -5,6 +5,8 @@
 {% load wagtailimages_tags %}
 
 {% block title %}{% trans "MIT Micromasters" %}{% endblock %}
```
Comment (pdpinch) [inline cms/templates/cms/program_page.html]: I'm not sure if I'm reading this correctly, but should we set this to be `{{ page.program.title }}` as well?

Comment (noisecapella) [inline cms/templates/cms/program_page.html]: This block is for the `<title>...</title>` element. It's probably a good idea to use `{{ page.program.title }}` there

File: cms/templates/cms/home_page.html [hunk 2]
```diff
@@ -7,6 +7,12 @@
 {% block body_class %}template-programpage{% endblock %}
 
 {% block title %}{% trans "MIT MicroMasters" %}{% endblock %}
+{% block description %}
+  The MicroMasters credential is a new option for online learning
+  in which the students can earn a full master's degree by taking
+  about half the course content online and half on campus.
```
Comment (pdpinch) [inline cms/templates/cms/home_page.html]: Here's the most recent copy from https://goo.gl/XFo3Fn

```
The MicroMasters is a new digital credential for online learners. The MicroMasters credential will be granted by MITx (MIT’s online learning initiative) to learners who excel in an integrated set of graduate-level online courses and a final capstone exam.  
```

@roberthouse54 do we have final copy yet?

If not, we should just go with this so we can move on.

File: cms/templates/cms/program_page.html (hunk 1 above)
Comment (annagav) [inline cms/templates/cms/program_page.html]: Why not `{{ page.title }}`? or `{{ page.title }} MicroMasters` to be consistent with the actual content of the page.

File: cms/templates/cms/home_page.html (hunk 2 above)
Comment (annagav) [inline cms/templates/cms/home_page.html]: I updated the text with this version.

File: cms/templates/cms/program_page.html (hunk 1 above)
Comment (noisecapella) [inline cms/templates/cms/program_page.html]: Yes you're right, we should use `page.title`

**Comment by noisecapella** [comment]
:+1:

File: cms/templates/cms/home_page.html [hunk 3]
```diff
@@ -7,6 +7,13 @@
 {% block body_class %}template-programpage{% endblock %}
 
 {% block title %}{% trans "MIT MicroMasters" %}{% endblock %}
+{% block description %}
+  The MicroMasters is a new digital credential for online learners. The MicroMasters
+  credential will be granted by MITx (MIT’s online learning initiative)
+  to learners who excel in an integrated set of graduate-level online courses
+  and a final capstone exam.
```
Comment (pdpinch) [inline cms/templates/cms/home_page.html]: Use this please (from @ferdi)

```
The MicroMasters is a new digital credential for online learners.  The 
MicroMasters credential will be granted to learners who complete an
integrated set of graduate-level online courses.  With the MicroMasters
credentials, learners can apply for an accelerated master's degree
program on campus, at MIT or other top Universities.
```

File: cms/templates/cms/home_page.html [hunk 4]
```diff
@@ -45,7 +52,7 @@
       <div class="banner-wrapper">
         <div class="banner-wrapper-content">
           <h1>
-            Learn. Challenge Yourself. Advance
+            MIT MicroMasters
```
Comment (pdpinch) [inline cms/templates/cms/home_page.html]: Can we change this copy while we're at it?

`Learn now, apply later`

File: cms/templates/cms/home_page.html [hunk 5]
```diff
@@ -45,7 +52,7 @@
...
           </h1>
           <h2>
             Top ranked programs from the #1 university in the world now delivered online. Advance your career with a fast-track master’s degree from MIT.
```
Comment (pdpinch) [inline cms/templates/cms/home_page.html]: And new copy here:

`Micromasters: A new digital credential for online learners`

Comment (annagav) [inline cms/templates/cms/home_page.html]: @roberthouse54 wanted to have `h1` and `<title>` the same. Maybe we should discuss this.

File: cms/templates/cms/home_page.html (hunk 3 above)
Comment (annagav) [inline cms/templates/cms/home_page.html]: Is this just for the header or should I change this in the body as well?

**Comment by roberthouse54** [comment]
This is just for the meta description tag in the header. The meta title tag in the header should be "MIT MicroMasters" and the H1 Header on the page will also be MIT MicroMasters.

File: cms/templates/cms/home_page.html (hunk 3 above)
Comment (annagav) [inline cms/templates/cms/home_page.html]: Rob said only in the header.

File: cms/templates/cms/home_page.html (hunk 4 above)
Comment (annagav) [inline cms/templates/cms/home_page.html]: `Micromasters: Learn now, apply later` is this correct?

**Comment by annagav** [comment]
screen shot 2016-09-27 at 12 49 27 pm

File: cms/templates/cms/home_page.html (hunk 4 above)
Comment (Ferdi) [inline cms/templates/cms/home_page.html]: MicroMasters: Learn now, apply later (or never)

**Comment by annagav** [comment]
Updated the title of the home page, and the description meta tag with the new text from #1140

**Comment by annagav** [comment]
screen shot 2016-09-27 at 4 42 14 pm

**Comment by pdpinch** [comment]
I'm really like the copy now. 

We just need some better contrast with the image so you can read it -- but that will be another PR. 

👍

**Pull request merged**